apps/nextjs-website/src/components/organisms/ChatbotHistoryDetailLayout/ChatbotHistoryDetailLayout.tsx

```diff
@@ -39,9 +50,18 @@ const ChatbotHistoryDetailLayout = ({
   nextSession,
   onDeleteChatSession,
 }: ChatbotHistoryDetailLayoutProps) => {
+  const [open, setOpen] = React.useState(false);
```

Comment:
You could simplify this line
```suggestion
  const [open, setOpen] = useState(false);
```
by importing 
`import { useState } from 'react';`